harness/setup.py

```diff
@@ -29,6 +29,7 @@
         "filelock",
         "google-cloud-storage",
         "hdfs>=2.2.2",
+        "jsonschema",
```

Comment:
so to be clear, this is something we already depended on, indirectly, only now we are less strict about the version overall, and also it's a dependency we're going to remove in the nearish future, right?